owners/scripts/warm_cache.js

```diff
@@ -32,9 +32,9 @@ require('dotenv').config();
 const sleep = require('sleep-promise');
 const Octokit = require('@octokit/rest');
 
-const {GitHub} = require('../src/github');
-const {VirtualRepository} = require('../src/repo');
-const {CompoundCache} = require('../src/file_cache');
+const {GitHub} = require('../src/api/github');
+const VirtualRepository = require('../src/repo/virtual_repo');
+const CompoundCache = require('../src//cache/compound_cache');
```

Comment:
Any reason why some of these use an object for `module.exports` while others don't?

Reply:
Because I haven't finished migrating them all to object exports imports yet :) I've been transitioning piecemeal to reduce merge/rebase hell

Reply:
Currently the `github` module exports classes for PRs, teams, and reviews, so it hasn't be updated like repositories and caches have